ql/src/test/queries/clientpositive/udf_trunc_number.q

```diff
@@ -71,4 +71,84 @@ DROP TABLE sampletable2;
 
 DROP TABLE sampletable3;
 
-DROP TABLE sampletable4;
\ No newline at end of file
+DROP TABLE sampletable4;
```

Comment:
Wouldn't be more appropriate to add unit tests (similar to `TestGenericUDFTrunc`) instead of qtests? The changes in this PR touch a very specific part of the code so I am not sure we need to add end-to-end tests.